modules/src/ics02_client/handler/create_client.rs

```diff
@@ -62,24 +63,29 @@ pub fn keep(keeper: &mut dyn ClientKeeper, result: CreateClientResult) -> Result
 mod tests {
     use super::*;
     use crate::ics02_client::context_mock::MockClientContext;
+    use crate::ics03_connection::msgs::test_util::get_dummy_account_id;
+    use crate::ics07_tendermint::client_state::ClientState;
     use crate::ics07_tendermint::header::test_util::get_dummy_header;
-    use crate::ics07_tendermint::msgs::create_client::MsgCreateClient;
     use crate::mock_client::header::MockHeader;
     use crate::mock_client::state::{MockClientState, MockConsensusState};
-    use std::str::FromStr;
+    use std::str::{from_utf8, FromStr};
     use std::time::Duration;
+    use tendermint::account::Id as AccountId;
     use tendermint::block::Height;
 
     #[test]
     fn test_create_client_ok() {
         let client_id: ClientId = "mockclient".parse().unwrap();
         let ctx = MockClientContext::default();
 
+        let signer = AccountId::from_str(from_utf8(&get_dummy_account_id()).unwrap()).unwrap();
```

Comment:
See above.

```suggestion
        let signer = AccountId::from_str(str::from_utf8(&get_dummy_account_id()).unwrap()).unwrap();
```

But since this seems to be a common pattern (at least in tests), perhaps that warrants having a `from_utf8` static method on `AccountId`? This would get rid of the two unwraps as well:

```suggestion
        let signer = AccountId::from_utf8(&get_dummy_account_id()).unwrap();
```